module/programs/helper.js

```diff
@@ -7,1058 +7,993 @@
 
 // Dependencies
 
-const timeZoneDifference = process.env.TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC;
-const programsQueries = require(DB_QUERY_BASE_PATH + "/programs")
-const entitiesService = require(GENERICS_FILES_PATH + "/services/entity-management")
+const timeZoneDifference = process.env.TIMEZONE_DIFFRENECE_BETWEEN_LOCAL_TIME_AND_UTC
```

Comment:
@Prajwal17Tunerlabs  what is the change in this file?

Reply:
the mongodb dump given by priyanka had "isReusable" field in a programs document, but in the model that particular field is not defined , until now the code was using "isReusable" and since this field was present in the dump , it didnt throw any error to me, but in the dev testing i discovered this field is not defined in the programs model!!

Reply:
yes @VISHNUDAS-tunerlabs  i checked in /programs/helper  it is not used anywhere, that was the only place it was used